src/components/IngredientCard/IngredientCard.scss

```diff
@@ -0,0 +1,53 @@
+.ingredient-card {
+    display: flex;
+    flex-direction: column;
+    width: 9em;
+    height: 8em;
+    border-radius: 0.6em;
+    overflow: hidden;
+    padding: 0.6em;
+    box-shadow: 0 0.4em 0.8em 0 rgba(0,0,0,0.2);
+
+    .ingredient-header {
+        display: flex;
+        justify-content: start;
+        width: 100%;
+        height: 40%;
+
+        .ingredient-image {
+            display: inline-block;
```

Comment:
You can make this a span to achieve this

Reply:
Talked offline. Comment in the wrong place.